### ISSUE TITLE
DCO check stays as `expected`

### ISSUE THREAD

**Issue by frouioui** [dcoapp/app#211]
Although the DCO app is installed in the org/user and on the repository, the check is no longer running for Pull Requests, the check stays as `expected`. This worked fine about an hour ago, and suddenly stopped working.

Is anyone experiencing this too?

EDIT: for those blocked by this issue @wagoodman proposed a nice solution (https://github.com/anchore/syft/pull/2926)

**Comment by zkoppert** [comment]
Yes, others experiencing this as well. It is a hosting side issue and I'm looking at it now.

**Comment by frouioui** [comment]
Thanks for the update @zkoppert.

**Comment by JeyJeyGao** [comment]
We are also afacted: https://github.com/notaryproject/notation/pull/952.
Our release process was blocked.

**Comment by wagoodman** [comment]
We're unable to wait until next week for a fix so have opted to switch to a different tool in the meantime until there is a functional deployment. For anyone else that is also in a bind and are looking for a simple workaround in the meantime [here's the PR](https://github.com/anchore/syft/pull/2926). 

We'll check back here next week for status on the deployment 👍

**Comment by zkoppert** [comment]
Hello again! I'm working to restore the service right now. Would be great to have folks comment here when they see service restored for confirmation that it is working.

**Comment by zkoppert** [comment]
Update: based on log data traffic appears to be restored to the service. Keeping this issue open until a 3rd party can confirm proper operation.

**Comment by zkoppert** [comment]
Confirmed with Linux Foundation the app is working as expected.

In order to trigger a recheck on an existing pull request you may need to close and reopen the pull request or push a new commit. 

Please reopen or open a new issue if you experience any problem.

**Comment by mihaimaruseac** [comment]
Thank you @zkoppert for looking at this.

I tried the close/open suggestion in https://github.com/sigstore/model-transparency/pull/195 but it doesn't seem to work. Will try pushing a new commit to see if that changes things.

**Comment by zkoppert** [comment]
Thank you for mentioning that. The above were my best guesses. :)

**Comment by mihaimaruseac** [comment]
Pushing an empty commit on https://github.com/sigstore/model-transparency/pull/190 has worked. 

Thank you.

**Comment by rbarkerSL** [comment]
I've been able to progress PRs waiting on DCO again in the last 30 minutes or so. 👍

**Comment by mynameistechno** [comment]
Thanks @zkoppert !
